lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -106,33 +109,33 @@ class window.Turbolinks
 
     return
 
-  @loadPage: (url, xhr, partialReplace = false, onLoadFunction = (->), replaceContents = [], replaceAllExcept = []) ->
+  @loadPage: (url, xhr, options = {}) ->
```

Comment:
I find they are less options, and more of the core signature of this method. So I prefer to leave it like this.